storage/backup/backup-cli/src/metadata/mod.rs

```diff
@@ -155,6 +162,13 @@ impl Metadata {
                 format!("transaction_{}-{}.meta", t.first_version, t.last_version)
             },
             Metadata::Identity(_) => "identity.meta".into(),
+            Self::CompactionTimestamps(e) => {
+                format!(
+                    "{}_{}.meta",
+                    Self::COMPACTION_TIMESTAMPS_FILE_NAME_PREFIX,
```

Comment:
the existence of this looks asymmetric to the surrounding, but I guess I'm fine.